src/renderer/js/component/link/view.jsx

```diff
@@ -41,16 +44,22 @@ const Link = props => {
     );
   }
 
+  // An acutal button is needed to use type="submit" for form submission
+  // this should probably just go off an "href" prop
+  // if we do that we will need to check all of the buttons that aren't supposed to look like buttons
+  const WrapperEl = actualButton ? "button" : "a";
```

Comment:
We have a `<SubmitButton>` component used elsewhere. Please use that instead.

(This code also has a styling error: the text weight when rendered as a button is less than that when rendered as an anchor.)